analysis/hostMicrobeInteractions/tutorial_hostMicrobeInteractions.m

```diff
@@ -21,19 +21,18 @@
 % can be downloaded from <https://wwwen.uni.lu/content/download/72950/917509/file/Mus_musculus_iSS1393.zip 
 % https://wwwen.uni.lu/content/download/72950/917509/file/Mus_musculus_iSS1393.zip>.
 %% 
+% change directory to where the tutorial is located
```

Comment:
The directory to the tutorial folder can only be changed **after** running `initCobraToolbox`. Otherwise, another `initCobraToolbox` that is already on the path will be used.